curriculumBuilder/settings.py

```diff
@@ -127,20 +127,29 @@
 
 # Supported languages
 LANGUAGES = (
+    ('ar-sa', _('Arabic')),
+    ('de-de', _('German')),
     ('en-us', _('English')),
     ('es-mx', _('Mexican Spanish')),
+    ('fr-fr', _('French')),
+    ('hi-in', _('Hindi')),
     ('it-it', _('Italian')),
-    ('th-th', _('Thai')),
```

Comment:
I arranged the languages into alphabetical order by language code, which is why there are "deleted" lines